docs/use-cases/resources.md

```diff
@@ -0,0 +1,124 @@
+
+* `s3`
+* `sns`
+* `sqs`
+* `syslog`
+* `webhook`
+* `memcached` *
+* `mysql` *
+* `postgres` *
+* `redis` *
+
+<div class="block-callout block-show-callout type-info" markdown="1">
+ * Unless required at the Rack level, we would recommend using these types at the App level.
+</div>
+
+
+### Creating a Rack Resource
+
+You can create a Rack resource very simply with the default options or you can pass in your optional values at creation time.
+
+```
+$ convox rack resources create postgres
+Creating resource... OK, postgres-8458
+
+$ convox rack resources create postgres MultiAZ=true
+Creating resource... OK, postgres-2871
+```
+
+### Listing Current Rack Resources
+
+```
+$ convox rack resources
+NAME                  TYPE     STATUS
+console-175092fe6ab1  webhook  running
+syslog-2984           syslog   running
+postgres-8458         postgres running
+postgres-2871         postgres running
+```
+
+#### Options
+
+You can see the available options for a resource type using the CLI:
+
+```
+$ convox rack resources options postgres
+NAME                        DEFAULT      DESCRIPTION
+AllocatedStorage            10           Allocated storage size (GB)
+BackupRetentionPeriod       1            The automatic RDS backup retention period, (default 1 day)
+Database                    app          Default database name
+Encrypted                   false        Encrypt database with KMS
+EngineVersion               9.6          Version of Postgres
+Family                      postgres9.6  Postgres version family
+InstanceType                db.t2.micro  Instance class for database nodes
+MultiAZ                     false        Multiple availability zone
+Password                    (generated)  Server password
+Username                    postgres     Server username
+```
+
+### Updating a Rack Resource
+
+```
+$ convox rack resources update postgres-8458 MultiAZ=true
+```
+
+### Deleting a Rack Resource
+
+```
+$ convox rack resources delete postgres-8458
+Deleting postgres-8458... OK
+```
+
+## Use Cases
+
+[Syslog](/deployment/syslogs) resources are widely used to connect your application logging to external 3rd party services.
+Internally, Convox uses the `webhook` resource to enable Slack integration.  You can do similar to receive [notifications](/console/notifications) about important events happening within your apps and Rack.  
+
+
+
+
+# External Resources
+
+For any resource types that Convox does not natively support, it is still very easy to integrate them into your infrastructure and environments.  
+Best practice is to inject an environment variable for each external resource that you wish to integrate.  
+
+For example if you needed a MariaDB instance, you would go create that in RDS, ensure that the VPC and Security Group allows access to it from your Rack, get the appropriate endpoint, then in the application that needs access, set the env var for your connection string:
```

Comment:
This section could use a lot more elaboration as this is the tricky part. How can they make sure it's accessible from their Rack?